textproc/lout/Portfile

```diff
@@ -1,28 +1,29 @@
-PortSystem 1.0
-PortGroup  compiler_blacklist_versions 1.0
-
-name            lout
-version         3.40
-license         GPL-3+
-categories      textproc
-platforms       darwin freebsd
-maintainers     gmail.com:ggreif
-description     A text formatting system similar to TeX, but simpler
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        william8000 lout 3.42
+github.tarball_from archive
+revision            0
+license             GPL-3+
+categories          textproc
+platforms           darwin freebsd
+maintainers         gmail.com:ggreif
```

Comment:
If possible, we should get a github handle for the maintainer (who probably has no idea of this pull request's existence.)

Reply:
Maybe it's Elias Pipping. I see his [commit in 2007](https://github.com/macports/macports-ports/commit/e534d63be18213ecf984b2c1579cbd522d968dd3#diff-a577ed9479e91af4a73e0525259550206cb1cce1312d2f54f7b113ce8ad89a69), when the port was created. I wrote to him at the indicated email.

Reply:
The old maintainer is me, happy to hand it over!

Reply:
Good, thanks for the quick reply!